tests/src/main/java/net/neoforged/neoforge/debug/data/DataMapTests.java

```diff
@@ -302,6 +306,40 @@ protected void gather(HolderLookup.Provider provider) {
         });
     }
 
+    @GameTest
+    @EmptyTemplate
+    @TestHolder(description = "Tests if data maps can be successfully attached to reloadable registries")
+    static void reloadableRegDataMaps(final DynamicTest test, final RegistrationHelper reg) {
```

Comment:
How does this gametest verify that resources are loaded only once for gametest servers?

Reply:
It does not
> This PR also adds a test to make sure that reloadable regs support data maps (there were no code changes needed, this test is just to make sure it doesn't break in the future)